clientv3/client.go

```diff
@@ -180,6 +188,15 @@ func newClient(cfg *Config) (*Client, error) {
 	client.Watcher = NewWatcher(client)
 	client.Auth = NewAuth(client)
 	client.Maintenance = &maintenance{c: client}
+	if cfg.Logger == nil {
+		// same as default std logger
+		client.logger = log.New(os.Stderr, "", log.Ldate|log.Ltime)
```

Comment:
i am not sure why we need to set client.logger in this case. if we want to set, it should be log.New(ioutil.Discard, "", 0).